pkg/sink/mysql/config.go

```diff
@@ -75,10 +75,12 @@ const (
 
 	// defaultcachePrepStmts is the default value of cachePrepStmts
 	defaultCachePrepStmts = true
-	// defaultStmtCacheSize is the default size of prepared statement cache
-	defaultPrepStmtCacheSize = 10000
+	// defaultPrepStmtCacheSize is the default size of prepared statement cache
+	// default size = (default max_prepared_stmt_count) / (default worker count + 1)
+	// 16382 / (16 + 1) = 963
+	defaultPrepStmtCacheSize = 963
```

Comment:
How does `worker count` affect the prepare statement cache size, what happens if user sets a larger `worker-count` in sink-uri.

Reply:
If the work-count increases, the number of prepared statements on the MySQL server will also increase, which may result in hitting the limit. The default size is set to ensure that there are no errors with a single changefeed and default work-count. However, if the user needs to increase the worker count or prepared statement cache size, they should also increase the max_prepared_stmt_count on the MySQL server. It's worth noting that TiDB does not have such a limit by default.

Reply:
`worker-count` is often set to a larger value, the risk of this error exists if `max_prepared_stmt_count` is not increased in TiDB/MySQL server.
Use a default value maybe not an ideal solution, is it possible to query `max_prepared_stmt_count` from downstream database and set a proper value of `prep-stmt-cache-size` based on the `max_prepared_stmt_count` and `worker-count`.

Reply:
hmm, make sense, I've give it a try.